### ISSUE TITLE
Failed to compile shader: "brush_mask"

### ISSUE THREAD

**Issue by NiLSPACE** [servo/servo#18974]
Servo just started throwing the following error in my console:

```
Failed to compile shader: "brush_mask"
ERROR: 0:1183: 'constructor' :  cannot convert parameter 1 from 'float' to 'structure'
ERROR: 0:1182: '=' :  cannot convert from '2-component vector of highp float' to 'highp structure'
```

Windows 10 just updated to the Fall Creators update, so it might have something to do with that.

**Comment by jdm** [comment]
cc @glennw

**Comment by NiLSPACE** [comment]
I don't get this error anymore.

**Comment by jdm** [comment]
Could you file this in https://github.com/servo/webrender/issues/? It will receive more attention there from the people who can fix it.

**Comment by NiLSPACE** [comment]
This has been fixed with #19054
